src/CONST.ts

```diff
@@ -985,6 +985,21 @@ const CONST = {
 
     TOOLTIP_MAX_LINES: 3,
 
+    TAXES: {
+        DEFAULT: {
+            EXTERNAL_ID: 'id_TAX_EXEMPT',
+            FOREIGN_ID: 'id_TAX_EXEMPT',
+            VALUE: '0%',
+            EXEMPT_ID: 'id_TAX_EXEMPT',
+            NAME: 'Tax',
+            EXEMPT_NAME: 'Tax exempt',
+            EXEMPT_VALUE: '0%',
+            TAX_RATE_1_ID: 'id_TAX_RATE_1',
+            TAX_RATE_1_NAME: 'Tax Rate 1',
+            TAX_RATE_1_VALUE: '5%',
+        },
+    },
+
```

Comment:
These data are obtained when I try turning on the tax feature in OD

Reply:
```
{ "defaultExternalID": "id_TAX_EXEMPT", "defaultValue": "0%", "foreignTaxDefault": "id_TAX_EXEMPT", "name": "Tax", "taxes": { "id_TAX_EXEMPT": { "name": "Tax exempt", "value": "0%" }, "id_TAX_RATE_1": { "name": "Tax rate 1", "value": "5%" } } }
```

Reply:
I left a comment above, I think the only difference is `id_TAX_OPTION_1` should be `id_TAX_RATE_1` and same for the name `Tax option 1` -> `Tax rate 1`

Reply:
> I left a comment above, I think the only difference is `id_TAX_OPTION_1` should be `id_TAX_RATE_1` and same for the name `Tax option 1` -> `Tax rate 1`

I think the code has incorporated most of these already. Only one minor change is pending: `Tax Rate 1` -> `Tax rate 1`